src/schemas/json/template.json

```diff
@@ -278,62 +380,234 @@
       "type": "array",
       "items": {
         "type": "object",
-        "required": [ "actionId", "manualInstructions" ],
+        "allOf": [
+          {
+            "required": [ "actionId", "manualInstructions" ],
 
-        "properties": {
-          "actionId": {
-            "description": "A guid uniquely defining the action. The value must correspond to a post-action known by the broker.",
-            "allOf": [ { "$ref": "#/definitions/guid" } ]
-          },
-          "args": {
-            "description": "A list of key-value pairs to use when performing the action. The specific parameters required / allowed are defined by the action itself.",
-            "type": "object",
-            "additionalProperties": {
-              "type": "string"
+            "properties": {
+              "actionId": {
+                "description": "A guid uniquely defining the action. The value must correspond to a post-action known by the broker.",
```

Comment:
Possibly add something like:
`When the actionId is not known to the broker, the behavior is defined by the broker. For example, both 'dotnet new' and Visual Studio will quietly ignore unknown actionIds'`